packages/components/Input/src/InputTokens.ios.ts

```diff
@@ -0,0 +1,86 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { InputTokens } from './Input.types';
+
+export const defaultInputTokens: TokenSettings<InputTokens, Theme> = (t: Theme) =>
+  ({
+    iconSize: 24,
+    hasIcon: {
+      //spacingIconContent + iconSize
+      spacingAssistiveTextStart: globalTokens.size160 + 24,
+      spacingLabelStart: globalTokens.size160 + 24,
```

Comment:
Should we move 24 to a constant?

Reply:
All places that use icon size currently use hard coded values

Reply:
Should possibly be replaced everywhere though, not including it here